.github/workflows/trigger-metamask-desktop-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Trigger MetaMask Desktop CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  trigger-desktop-ci:
+    runs-on: ubuntu-latest
+    environment: desktop
```

Comment:
This is just a suggestion. To be updated once the environment is created.

Reply:
Sounds good to me! I've setup the environment with the required secrets